backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/storage/disk/image/TransferDiskImageCommand.java

```diff
@@ -535,6 +535,11 @@ public void proceedCommandExecution(Guid childCmdId) {
             setImageGroupId(entity.getDiskId());
         }
 
+        if (isHybridBackup() && entity.getPhase().canBeCancelled() && getBackup().isStopped()) {
+            updateEntityPhase(ImageTransferPhase.CANCELLED_USER);
+            return;
```

Comment:
why not continue to executeStateHandler()?
I guess it will get there on the next polling iteration, but I'm wondering if there's a particular reason not to do it within the same iteration

Reply:
I've seen that the entity object is not updated after the entity is updated in the DB. So if we proceed to executeStateHandler, it enters `case TRANSFERRING` in this cycle anyway. Actually nothing wrong with that, but I thought it would be cleaner to do it this way.

Reply:
I see, makes sense because the value was already fetched before the was overwritten

anyway, I'd add a log here to make it clear that the transfer is canceled because of the backup

Reply:
Good point, done